test/module/irohad/torii/torii_service_test.cpp

```diff
@@ -142,6 +141,30 @@ class ToriiServiceTest : public testing::Test {
   std::shared_ptr<MockStatelessValidator> statelessValidatorMock;
 };
 
+
+/**
+ * @given chain of CommandClient copies and moves
+ * @when status is asked
+ * @then grpc returns ok
+ */
+TEST_F(ToriiServiceTest, CommandClient) {
+
+  iroha::protocol::TxStatusRequest tx_request;
```

Comment:
Use shared_model instead.

Reply:
Why?
The purpose of this test is to check the client

Reply:
Ok, we'll rework it in next time.